a/unittests/lbge-unittests.asd

```diff
@@ -0,0 +1,9 @@
+(defpackage :lbge-unittests-system
+  (:use :cl))
+
+(in-package :lbge-unittests-system)
+
+(asdf:defsystem :lbge-unittests
```

Comment:
IDK about `unittests` everywhere, maybe `unit-tests`?

Reply:
May be `unit-tests` as well.